src/Kestrel.Core/Internal/Http/Frame.cs

```diff
@@ -345,11 +345,11 @@ public void Reset()
             StatusCode = StatusCodes.Status200OK;
             ReasonPhrase = null;
 
-            RemoteIpAddress = RemoteEndPoint?.Address;
-            RemotePort = RemoteEndPoint?.Port ?? 0;
+            RemoteIpAddress = HttpConnectionFeature?.RemoteIpAddress;
+            RemotePort = HttpConnectionFeature?.RemotePort ?? 0;
```

Comment:
Do these need to be separate things (Address and Port) rather than `IPEndPoint`? As its moving from a 4x property lookups to 4x interface property lookups